src/components/OperationList.tsx

```diff
@@ -0,0 +1,171 @@
+import * as React from "react";
+import { connect, MapStateToProps } from "react-redux";
+
+import styled from "styled-components";
+import { green50 } from "material-ui/styles/colors";
+
+import { Uid } from "../identifiers";
+import { AppState } from "../store";
+import { Member } from "../reducers/membersNew";
+import { Operation, OperationType } from "../reducers/operations";
+import { getMembersByUid } from "../selectors/members";
+
+import IntlMessage from "./IntlMessage";
+
+// styling from: https://material-components.github.io/material-components-web-catalog/#/component/list
+const OperationList = styled.ul`
+  padding: 8px 0px;
+  background-color: ${green50};
+  border: 1px solid rgba(0, 0, 0, 0.1);
+
+  > li {
```

Comment:
Why do we need the .operation_text element? It’s styles can just be on the Li element and remove the spam altogether, it doesn’t express anything

Reply:
👍